middleware/gateway/include/gateway/manager/admin/model.h

```diff
@@ -82,6 +83,7 @@ namespace casual
                   case Runlevel::connecting: return "connecting";
                   case Runlevel::pending: return "pending";
                   case Runlevel::connected: return "connected";
+                  case Runlevel::disconnecting: return "disconnecting";
```

Comment:
Not related to this commit, but we should have some kind of macro (or such) for an enum-to-string-thing

Reply:
Well, it depends on the _macro_. But I don't think the benefits are that good. It's not hard to write the `description`, and the compiler will tell you if you forgot values, or values are added over time.
Hence, the benefits are only convenience at best, and not to help/force doing the correct thing, since this is covered by the compiler.
But you might think of a _macro_ that does some "magical introspection" stuff?